woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -791,10 +790,16 @@ def test_ordinal_with_incomplete_ranking(sample_series):
     ordinal_incomplete_order = Ordinal(order=['a', 'b'])
     error_msg = re.escape("Ordinal column sample_series contains values that are not "
                           "present in the order values provided: ['c']")
+
+    schema_df = pd.DataFrame(sample_series)
+
     with pytest.raises(ValueError, match=error_msg):
-        schema_df = pd.DataFrame(sample_series)
         schema_df.ww.init(logical_types={'sample_series': ordinal_incomplete_order})
 
+    with pytest.raises(ValueError, match=error_msg):
+        schema_df.ww.init()
```

Comment:
Maybe move this line outside the context manager so we know clearly which statement we are testing. Not sure why the df creation was inside before...